CONTRIBUTING.md

```diff
@@ -24,6 +24,18 @@ If submitting a PR consisting of documentation changes only, please try to ensur
 
 The development team will be happy to help and guide you with any of these points and work with you getting your PR submitted for approval. Create a PR with [WIP] in the title and ask for specific assistance within the issue, or contact the dev team on any of the channels below.
 
+# Branching Model
+
+Grin's current branching model is as follows:
+
+`master` is used for development of the current point release. For instance, if the last released version of Grin is 1.2.0, master will contain changes intending to go into 1.2.1, 1.2.2 etc. Breaking and backwards-incompatible changes should not occur between point releases, and should not be merged into master.
+
```

Comment:
Is it expected that `master` always should be safe to build and be  deployable on production?

Reply:
Good question. Deployable on production I'd say no, it's expected to be bleeding edge and only releases should be deemed 'safe for production'. It should always build, however.

Reply:
Based on older discussions, the basic premise for the current branching model was that `master` is meant to be deployable (think of the simple case: clone+build+run) and breaking changes are developed as part of the milestone branches. If `master` is only meant to be buildable then what's the point of the milestone branches? I assume a milestone branch is also buildable if you get something merged into it, ie. it passes CI.